Sources/Commands/Workspace.swift

```diff
@@ -651,6 +739,8 @@ public class Workspace {
 
         // First, add the root package constraints.
         var constraints = computeRootPackageConstraints(currentManifests.root)
```

Comment:
Why not updated computeRootPackageConstraints to take an argument about whether or not to include pins? That will force all clients to think about it